docs/user-guide/helm.md

```diff
@@ -499,6 +499,26 @@ spec:
       skipCrds: true
 ```
 
+## Helm `--skip-schema-validation`
+
+Helm validations the values.yaml file using a values.schema.json file. See [Schema files](https://helm.sh/docs/topics/charts/#schema-files) for details.
```

Comment:
```suggestion
Helm validates the values.yaml file using a values.schema.json file. See [Schema files](https://helm.sh/docs/topics/charts/#schema-files) for details.
```

Reply:
I made the change, but when I try to run `make codegen` I get an error basically as if `make` or something that `make` is running is confined in some way? The errors are things like
* mkdir: cannot create directory '/go/src/github.com/gogo': Permission denied
* mkdir: cannot create directory '/go/src/k8s.io': Permission denied

And the project root is seemingly wrong as well? 
```
+++ dirname ./hack/generate-proto.sh
++ cd ./hack/..
++ pwd
+ PROJECT_ROOT=/go/src/github.com/argoproj/argo-cd
```

Running `make protogen-fast` by itself (the stage that is throwing the error) seems to work just fine though. Apologies if I am missing something basic, this is my first PR to argo.

Reply:
I was able to get the `make codegen` to run by forcing the `run-in-test-client` part to run as root in the container. This is not part of the PR, and the file generated so *shrug*? I think it should work at this point. The comment about fixing the help typo has been addressed